packages/kit/src/runtime/client/router.js

```diff
@@ -109,12 +109,13 @@ export class Router {
 		const handle_mousemove = (event) => {
 			clearTimeout(mousemove_timeout);
 			mousemove_timeout = setTimeout(() => {
-				trigger_prefetch(event);
+				event.target.dispatchEvent(new CustomEvent('trigger_prefetch', { bubbles: true }));
 			}, 20);
 		};
 
 		addEventListener('touchstart', trigger_prefetch);
 		addEventListener('mousemove', handle_mousemove);
+		addEventListener('trigger_prefetch', trigger_prefetch);
```

Comment:
I think we can namespace the event as `sveltekit:trigger_prefetch` so we know it's SvelteKit specific.